python_ta/transforms/type_inference_visitor.py

```diff
@@ -359,13 +366,13 @@ def visit_index(self, node: astroid.Index) -> None:
         node.inf_type = node.value.inf_type
 
     def visit_slice(self, node: astroid.Slice) -> None:
-        lower_type = node.lower.inf_type.getValue() if node.lower else type(None)
-        upper_type = node.upper.inf_type.getValue() if node.upper else type(None)
-        step_type = node.step.inf_type.getValue() if node.step else type(None)
+        lower_type = node.lower.inf_type if node.lower else type(None)
+        upper_type = node.upper.inf_type if node.upper else type(None)
+        step_type = node.step.inf_type if node.step else type(None)
         node.inf_type = self._handle_call(node, '__init__', slice, lower_type,
                                           upper_type, step_type)
-        if node.inf_type.getValue() == type(None):
-            node.inf_type = TypeInfo(slice)
+        node.inf_type = node.inf_type >> (
+            lambda t: TypeInfo(slice) if t == type(None) else TypeInfo(t))
```

Comment:
This logic is a bit funny and might have been resolved. Can you please check with @simeonkr to see if it's still possible for `self._handle_call` to return "NoneType" here?